### PR TITLE
Move autorest client to java sdk central repo

### PR DESCRIPTION
**Pull request by sima-zhu**
There are basically three steps moving the common runtime package to central repo:

1. First, move the entire package without changes.
2. Then, we will cleanup the pom file, spotbugs and checkstyle.
3. Last, we will run the live tests.

This commit is to finish the following first step.

issue:  Move autorest client to java sdk central repo #3174

### CODE REVIEW

**Comment by mssfang** [comment]
LGTM.

**Comment by JonathanGiles** [comment]
Sima, don't merge this - ask @weshaggard to do it so that the revision history is maintained.

**Comment by weshaggard** [comment]
@sima-zhu what is the issue tracking the remaining clean-up work? Your description says:

> issue: Move autorest client to java sdk central repo #3174

but that is this PR not the an issue. I want to make some notes about some things that need to be cleaned up.

File: common/versionConfig.csv [hunk 1]
```diff
@@ -0,0 +1,5 @@
+com.microsoft.azure.v3,autorest-clientruntime-for-java,2.0.0,2.0.1-SNAPSHOT
```
Comment (weshaggard) [inline common/versionConfig.csv]: We should understand if anything will be using this file and if not remove as part of clean-up.

File: common/build-tools/src/main/resources/suppressions.xml [hunk 2]
```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0"?>
```
Comment (weshaggard) [inline common/build-tools/src/main/resources/suppressions.xml]: As part of the clean-up these should be moved the suppression files under the eng directory.

File: common/build-tools/pom.xml [hunk 3]
```diff
@@ -0,0 +1,48 @@
+<!--
```
Comment (weshaggard) [inline common/build-tools/pom.xml]: This project should be removed during clean-up.

**Comment by sima-zhu** [comment]
Hello Wes,

I don’t have an issue tracking the remaining clean up work yet. Do we need to open an issue for the remaining clean up? I can do it right now.
Anu, could you please advise the best practice for this?
Thanks!

Regards,
Sima Zhu
Software Engineer
(607)-768-5356
Room 3960, Building 18

________________________________

File: common/.gitignore [hunk 4]
```diff
@@ -0,0 +1,31 @@
+build
```
Comment (weshaggard) [inline common/.gitignore]: This file should be removed during clean-up.

File: common/.travis.yml [hunk 5]
```diff
@@ -0,0 +1,17 @@
+language: android
```
Comment (weshaggard) [inline common/.travis.yml]: Should be removed during clean-up.

File: common/LICENSE [hunk 6]
```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
```
Comment (weshaggard) [inline common/LICENSE]: Should be removed during clean-up.

File: common/README.md [hunk 7]
```diff
@@ -0,0 +1,124 @@
+[![Build Status](https://travis-ci.org/Azure/autorest-clientruntime-for-java.svg?branch=v2)](https://travis-ci.org/Azure/autorest-clientruntime-for-java)
```
Comment (weshaggard) [inline common/README.md]: The README.md files should go under the package folder during clean-up.

File: common/azure-android-client-authentication/build.gradle [hunk 8]
```diff
@@ -0,0 +1,117 @@
+buildscript {
```
Comment (weshaggard) [inline common/azure-android-client-authentication/build.gradle]: We aren't using gradle to build so we should convert to maven projects during the clean-up.

**Comment by weshaggard** [comment]
>I don’t have an issue tracking the remaining clean up work yet. Do we need to open an issue for the remaining clean up? I can do it right now.

Yes please file a follow-up issue in this repo to track the remaining work that needs to be done.

**Comment by sima-zhu** [comment]
[Issue ](https://github.com/Azure/azure-sdk-for-java/issues/3198)tracking the remaining clean up.

**Pull request merged**